### PR TITLE
Michael/rox 9892 negate criteria

### PR DESCRIPTION
**Pull request by 0x656b694d**
## Description

A detailed explanation of the changes in your PR.

Feel free to remove this section if it is overkill for your PR, and the title of your PR is sufficiently descriptive.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [stackrox/openshift-docs](https://github.com/stackrox/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

TODO(replace-me)
Use this space to explain how you tested your PR, or, if you didn't test it, why
you did not do so. Valid reasons include, for example, "CI is sufficient",
"No testable changes". Feel free to attach JSON snippets, curl commands,
screenshots.

In addition to reviewing your code, reviewers **must** also review your testing
instructions and make sure they are sufficient.

### CODE REVIEW

File: pkg/booleanpolicy/augmentedobjs/construct.go [hunk 1]
```diff
@@ -216,42 +211,41 @@ func ConstructImage(image *storage.Image) (*pathutil.AugmentedObj, error) {
...
+				result.GetStatus().String(),
+				pathutil.FieldStep("SignatureVerificationData"),
+				pathutil.FieldStep("Results"),
+				pathutil.IndexStep(i),
+				pathutil.FieldStep("Status"),
+			)
+			if err != nil {
+				return nil, utils.Should(err)
 			}
 		}
```
Comment (md2119) [inline pkg/booleanpolicy/augmentedobjs/construct.go:237]: You do not need to do this. The image signature results field is already a part of image object. Here, you are reading it out and writing it back.

Augmented objects constructions must be used to attach new fields to an objects.

File: pkg/booleanpolicy/querybuilders/special_cases.go [hunk 2]
```diff
@@ -131,3 +131,32 @@ func mapFixedByValue(s string) string {
 	}
 	return valueToStringRegex(s)
 }
+
+// ForImageSignatureVerificationStatus returns a query builder for Image
+// Signature Verification Status.
+func ForImageSignatureVerificationStatus() QueryBuilder {
```
Comment (md2119) [inline pkg/booleanpolicy/querybuilders/special_cases.go:137]: This looks fine.

File: proto/storage/image.proto [hunk 3]
```diff
@@ -102,7 +102,7 @@ message ImageSignatureVerificationResult {
         // status.
         GENERIC_ERROR          = 5;
     }
-    Status status                               = 3;
+    Status status                               = 3 [(gogoproto.moretags) = "search:\"Image Signature Verification Status,store\""];
```
Comment (md2119) [inline proto/storage/image.proto:105]: same as above

**Comment by 0x656b694d** [comment]
https://github.com/stackrox/stackrox/pull/1185

**Pull request closed without merging**